src/app/components/battle.ts

```diff
@@ -0,0 +1,69 @@
+import { createElement } from '../utils/createElement'
+import { Audio } from '../audio'
+import { Game } from '../game'
+
+export class Battle {
+  private audio: Audio
+  private game: Game
+  private mainArea: HTMLElement = document.querySelector(
```

Comment:
Can we place all those elements(not Audio and Game) in _init() / getUIElements() or something like that and run this method in the constructor?